kubernetes/redpanda/templates/configmap.yaml

```diff
@@ -14,7 +16,7 @@ data:
         {{- range untilStep 0 (.Values.replicaCount|int) 1 }}
         - node_id: {{ . }}
           host:
-            address: {{ template "redpanda.name" $envAll }}-{{ . }}
+            address: {{ template "redpanda.name" $envAll }}-{{ . }}.{{ template "redpanda.name" $envAll }}
```

Comment:
Why is redpanda.name doubled up like this?

Reply:
Because the `satefulSet.serviceName` is templated, and thats the governing service name for DNS: https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#stable-network-id